test_regression.py

```diff
@@ -0,0 +1,157 @@
+
+#%%
+import pytest
+from pathlib import Path
+import numpy as np
+import os
+
+from utils import extract_test_geometries, execute_and_assert
+
+
+def test_apply_kernel(auth_connection, tmp_path):
+
+    # Define scenario parameters
+    # Load scenario parameters
+    scenario_name = 'apply_spatial_kernel'
+    
+    # Set up output directory and path
+    output_path = tmp_path / f'{scenario_name}.nc'
```

Comment:
FYI: the output file name does not have to be unique in some sense, your `tmp_path` work folder is guaranteed to be unique already (and will include references to the current test function in its dir name), so you can just use "output.nc".